kubernetes/linux/main.sh

```diff
@@ -581,8 +581,8 @@ fi
 
 # If the prometheus sidecar isn't doing anything then there's no need to run mdsd and telegraf in it.
 if [[ ( "${CONTAINER_TYPE}" == "PrometheusSidecar" ) &&
-      ( "${TELEMETRY_CUSTOM_PROM_MONITOR_PODS}" == "false" ) &&
-      ( "${TELEMETRY_OSM_CONFIGURATION_NAMESPACES_COUNT}" -eq 0 ) ]]; then
+      ( "${TELEMETRY_CUSTOM_PROM_MONITOR_PODS}" == "false"  || "${TELEMETRY_CUSTOM_PROM_MONITOR_PODS}" == "" ) &&
+      ( "${TELEMETRY_OSM_CONFIGURATION_NAMESPACES_COUNT}" -eq 0 || "${TELEMETRY_OSM_CONFIGURATION_NAMESPACES_COUNT}" == "" ) ]]; then
```

Comment:
can this be improved as well so that there is just 1 condition? is it possible to set default value for TELEMETRY_OSM_CONFIGURATION_NAMESPACES_COUNT in tomlparser-prom-customconfig.rb?

Reply:
Added default value of variable in main.sh